### DISCUSSION TITLE
Image path bug or is it me?

### DISCUSSION THREAD

**Discussion by ZeFish** [ObsidianPublisher/obsidian-github-publisher#250] [General]
Am I doing something wrong?
The folder don't seems to be fine

<img width="759" alt="Screenshot 2023-12-04 at 11 16 47 AM" src="https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/24323/81ee0bb1-89a2-41e7-ad41-5f39ee523e7f">

**Comment by Mara-Li** [comment]
Without any settings I can't do anything?
